core/trino-main/src/main/java/io/trino/operator/HttpPageBufferClient.java

```diff
@@ -486,7 +486,7 @@ public void onFailure(Throwable t)
                             t = new TrinoException(GENERIC_INTERNAL_ERROR, format("Checksum verification failure on %s when reading from %s: %s", selfAddress, uri, t.getMessage()), t);
                             break;
                         case RETRY:
-                            log.warn("Checksum verification failure on %s when reading from %s, may be retried: %s", selfAddress, uri, t.getMessage());
+                            log.debug("Checksum verification failure on %s when reading from %s, may be retried: %s", selfAddress, uri, t.getMessage());
```

Comment:
this as well is very useful because it signals a possible issue in the network and would just appear as longer execution times without any idea of the reason.